### DISCUSSION TITLE
是正版吗

### DISCUSSION THREAD

**Discussion by xhnsw** [share121/inter-knot#5064] [原神]
我下的原神是正版吗
![Screenshot_2024-08-21-17-27-12-58_f551421da3aa686ae5366a5f197642ce](https://github.com/user-attachments/assets/20d21e46-8004-448b-8914-d1771d5c4a6c)

**Comment by Wattloo** [comment]
柚子厨给我抄抄

**Reply by xhnsw** [reply]
孩子你无敌了

**Comment by March-777th** [comment]
包正版的<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by love262-wkx** [comment]
逆天

**Comment by cylca4869** [comment]
太正版了<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Comment by Akanyi** [comment]
太正版了，物极必反，以至于有点盗版。

**Comment by igific** [comment]
千原万神

**Comment by wshtdgo** [comment]
妈呀，包正版的啊
